### DISCUSSION TITLE
Add dropdown property editor default selection setting

### DISCUSSION THREAD

**Discussion by bkclerke** [umbraco/Umbraco-CMS#13588] [Features and ideas]
I think a helpful feature would be to add the setting on the [Dropdown property editor](https://docs.umbraco.com/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/dropdown) to select a default prevalue option.

For example when you are adding prevalue options to a select list, It would be nice to be able to have a default value selected to save content editors a step in selection. We can set defaults in code, however that is not reflected to the content editor when they see their selection dropdown list in the backoffice.

Example of concept:
![Screenshot 2022-12-15 at 1 29 07 PM](https://user-images.githubusercontent.com/4968254/207971075-66a53011-1744-4584-8670-3573d30fed48.png)

**Comment by lassefredslund** [comment]
Hi @bkclerke 
Thank you for this idea. I think you are right, and that it could be a nice addition to the Dropdown Property Editor (and maybe the Color Picker and RadioButtonList too).
I'll set the status as up for grabs.

**Comment by lassefredslund** [comment]
Maybe this is relevant for almost all Property Editors. It would therefore be nice if it could be implemented in a generic way so that it works similarly for all Property Editors.

**Comment by nul800sebastiaan** [comment]
Currently you should be able to do this with https://our.umbraco.com/documentation/reference/events/EditorModel-Events/ and https://docs.umbraco.com/umbraco-cms/reference/notifications/editormodel-notifications (Good blog post from Paul Seal as well: https://codeshare.co.uk/blog/how-to-add-default-values-to-umbraco-properties-in-the-backoffice/).

The reason we've never implemented default values for anything is because: yes - it's a BIG wish for us to have default values, for all property editors and that is a massive undertaking. 

I've added some details on how that should be achieved on this older issue here: https://github.com/umbraco/Umbraco-CMS/issues/7859

So I think the answer for now is: use the EditorModel and maybe we can facilitate something for this in the new backoffice build?
